ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -421,14 +421,15 @@ ocp_version=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '.' -f1,2)
 origin_index_image="quay.io/openshift-qe-optional-operators/aosqe-index:v${ocp_version}"
 mirror_index_image="${MIRROR_PROXY_REGISTRY_QUAY}/openshift-qe-optional-operators/aosqe-index:v${ocp_version}"
 
+create_settled_icsp
```

Comment:
It may not work.  I think we have to replace the URL registry.redhat.io with brew.registry.redhat.io before mirror.

Reply:
Where is this shell script running?  Can we install skopeo on the slave?

Reply:
In another direction, can we ssh into a machine in which skopeo can be used?  @yunjiang29

Reply:
Done, updated it.

Reply:
We don't have _registry-proxy_ in C2S/SC2S environment.

Reply:
The fix still won't work. Let meet tomorrow.

Reply:
This `oc adm catalog mirror` running on the slave, not the cluster, so the ICSP doesn't work. Do we really need those images? If not, we can skip it. If yes, I think we have to replace the URL registry.redhat.io with brew.registry.redhat.io before mirroring.
```
error: unable to retrieve source image registry.redhat.io/openshift-logging/cluster-logging-rhel9-operator manifest sha256:2801ca51e913273d7db6728f05cb50f18b5c0062f95e92609645b592512aed21: manifest unknown: manifest unknown
```